brightlayer-ui/angular-component-library
# Make it so drawers have their unique instance of state management

Fixes #453 


#### Changes proposed in this Pull Request:
- Makes it so that each time a new `DrawerComponent` is constructed, a new `DrawerState` is made and shared w/ all subsections & `DrawerLayout` parent.
- Removed all `openOnDrawerHover` overrides from doc examples
- Defaults `Drawer` `open` property (required) to true if not required.  This is convenient for our examples, otherwise we'd have to manually mark each drawer as open. 


#### To Test:
- `yarn start` and start hover on Nav Items.  Observe how the main navigation drawer doesn't open/close.
